R/utils-classes.R

```diff
@@ -399,11 +397,11 @@ collect.bcdc_promise <- function(x, ...){
     # Create pagination client
     cc <- crul::Paginator$new(
       client = cli,
-      by = "query_params",
+      by = "limit_offset",
       limit_param = "count",
       offset_param = "startIndex",
       limit = number_of_records,
-      chunk = getOption("bcdata.chunk-limit", default = 1000),
+      chunk = getOption("bcdata.chunk_limit", default = 1000),
```

Comment:
I agree with this change but we should definitely document it in the NEWS file as it technically is a breaking change.

Reply:
I don't think it is a breaking change, just a typo fix - it's [documented with an underscore](https://github.com/bcgov/bcdata/blob/d780f108de4d5a232d6a88ced6b00a6f3776a63f/R/bcdc_options.R#L68) - this was a typo!

Reply:
Added NEWS item in 79e2c601